notebooks/03.09-Pivot-Tables.ipynb

```diff
@@ -80,9 +80,9 @@
        "      <th>sex</th>\n",
```

Comment:
<p>Some more test comments for spacing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Reply to test spacing</p>

Reply:
<p>a</p>

Reply:
<p>test is important</p>

Reply:
<p>this is the reply </p><p> </p><p><a href="https://app.reviewnb.com/amit1rrr/handson-ml-private/pull/15#comment-diff-1335149342" target="_blank">https://app.reviewnb.com/amit1rrr/handson-ml-private/pull/15/#comment-diff-1335149342</a></p><p> </p><p>comment! </p>